src/sass/components/_how-we-do.scss

```diff
@@ -0,0 +1,151 @@
+@use '../utils/vars' as vars;
+
+.how-we-do__section {
+  list-style-type: none;
+  display: flex;
+  justify-content: center;
+}
+
+.how-we-do__heading {
+  font-family: 'Montserrat';
+  font-style: normal;
+  font-weight: 800;
+  font-size: 40px;
```

Comment:
Wielkość czcionki na mobile to 20px. Zmienia się ona wraz z rozmiarem strony na 26px przy tablecie i 46px na desktopie.